kasa/protocols/smartprotocol.py

```diff
@@ -54,6 +54,8 @@
     "mfi_setup_id": None,
     "mfi_token_token": None,
     "mfi_token_uuid": None,
+    # Vacuum
+    "map_data": None,
```

Comment:
`dump_devinfo` has it's own redactor in the module

Reply:
Fixed.